src/machines/java/com/enderio/machines/client/gui/widget/ExperienceWidget.java

```diff
@@ -51,19 +52,17 @@ public void renderWidget(GuiGraphics guiGraphics, int pMouseX, int pMouseY, floa
         guiGraphics.drawString(font, s, this.x + this.width/2, (float)(this.y - this.height - 3 - 1), 0, false);
         guiGraphics.drawString(font, s, this.x + this.width/2, (float)this.y - this.height - 3, 8453920, false);
 
-        RenderSystem.enableBlend();
-        RenderSystem.setShaderColor(1.0F, 1.0F, 1.0F, 1.0F);
-
         renderToolTip(guiGraphics, pMouseX, pMouseY);
```

Comment:
Maybe we just remove the renderTooltip method and do the isHovered check and setTooltip in here. Its not really "rendering" the tooltip anymore